app/javascript/components/dual-list-select/helpers.jsx

```diff
@@ -0,0 +1,29 @@
+export const isEmpty = (obj) => {
+  if (obj === '') { return true; }
+  return (Object.getOwnPropertyNames(obj).length === 0);
+};
+
+/* sorts alphabetically by the second values in arrays: [ ["value", "abc"], ["value1", "baa"], ... ] */
+export const comparator = (a, b) => {
```

Comment:
take a look at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare pls

Reply:
Fixed! :+1: 

(For last 10 minutes I was trying to figure out why localCompare is not a function (instead of localeCompare :smile:  ))